tests/integration/datadict/submission/test_admin_endpoints.py

```diff
@@ -94,63 +94,61 @@ def test_to_delete(
         assert sur_node.sysan.get("to_delete") is to_delete
 
 
-def do_reassign(client, headers):
-    """Perform the http reassign action
-
-    Args:
-        client (pytest.Fixture): Allows you to mock http requests through flask
-        headers (dict): http headers with token
+def test_reassign(
+    client, pg_driver, cgci_blgsp, index_client, submitter, require_index_exists_off
+):
+    """Try to reassign a node's remote URL
 
-    Returns:
-        requests.Response: http response from doing reassign request
-        string: did
-        string: s3 url that you changed it to
+    Url:
+        PUT: /admin/<program>/<project>/files/<file_uuid>/reassign
```

Comment:
this (updating URLs) should be supported in indexd, not sheepdog... and this code only supports S3 URLs, and it assumes there is only 1 stored URL 🤦‍♀ we should probably get rid of this and add an endpoint in indexd instead

cc @Avantol13 re:conversation about indexd not supporting URL updates

Reply:
Should we wait until indexd has the endpoint or shall I go ahead and remove this~

Reply:
we can wait to be safe, though I don't think it's actually used anywhere? but if we change the public API we need a new major version of sheepdog